css/index.css

```diff
@@ -104,14 +132,53 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
For each of the boxes you don't need to target .middle-content .boxes .box1.  You can target just the .box1, .box2 .box3 etc.